posthog/models/team/team.py

```diff
@@ -276,6 +282,9 @@ class Meta:
     person_display_name_properties: ArrayField = ArrayField(models.CharField(max_length=400), null=True, blank=True)
     live_events_columns: ArrayField = ArrayField(models.TextField(), null=True, blank=True)
     recording_domains: ArrayField = ArrayField(models.CharField(max_length=200, null=True), blank=True, null=True)
+    cookieless_server_hash_mode = models.SmallIntegerField(
+        default=CookielessServerHashMode.DISABLED, choices=CookielessServerHashMode.choices, null=True
+    )
```

Comment:
Why make it nullable? What Postgres version are we running?

If we are on PG 11+ we can make it non-nullable, and because there's a default it will automatically "backfill" it in a lightweight manner (i.e. it won't backfill, it computes it when returning the row)

https://git.postgresql.org/gitweb/?p=postgresql.git;a=commit;h=16828d5c0273b4fe5f10f42588005f16b415b2d8

Reply:
The problem with making it nullable is that we now need to handle `DISABLED`, `STATEFUL`, `STATELESS` and now `null` as well (which we aren't)

Reply:
`SELECT version();`

> PostgreSQL 15.4 on aarch64-unknown-linux-gnu, compiled by aarch64-unknown-linux-gnu-gcc (GCC) 9.5.0, 64-

Reply:
So yes I should be able to use this if I can make django do the right thing

Reply:
Well DISABLED and null are just treated as the same atm (as both python and JS will treat them as equally falsy)

Reply:
> Well DISABLED and null are just treated as the same atm (as both python and JS will treat them as equally falsy)
True, but is that a coincidence or a conscious choice? Similarly, this only true until we need to read this from Rust.

Anyway, this is going to work for now, and it is minor, but I still think that we should eventually move to a world where fields are not unconsciously nullable if they don't need to be

Reply:
I'm on it :)